lib/internal/url.js

```diff
@@ -1460,14 +1460,14 @@ function getPathFromURLPosix(url) {
   return decodeURIComponent(pathname);
 }
 
-function fileURLToPath(path) {
+function fileURLToPath(path, { windows } = {}) {
```

Comment:
The use of destructuring on the argument here is a bit problematic if the user happens to pass `undefined` for the `options`. This should likely just be passed as `options = {}` with a check to verify that `option` is an object followed by pulling out the `windows` option.